java/openjdk10/Portfile

```diff
@@ -4,29 +4,48 @@ PortSystem       1.0
 
 name             openjdk10
 version          10.0.2
+set              openjdk_version 10
+
+subport openjdk11 {
+    version      11
+    set          openjdk_version 11
+    set          build 28
+}
 
 categories       java devel
 maintainers      {breun.nl:nils @breun} openmaintainer
 platforms        darwin
 license          GPL-2
 supported_archs  x86_64
 
-description      Open Java Development Kit 10
+description      Open Java Development Kit ${openjdk_version}
 
 long_description Production-ready, free and open-source build of the Java \
                  Development Kit, an implementation of the Java Standard \
-                 Edition (SE) 10 Platform. OpenJDK is the official reference \
+                 Edition (SE) ${openjdk_version} Platform. OpenJDK is the official reference \
                  implementation of Java SE. Included components are the \
                  HotSpot virtual machine, the Java class library and the Java \
                  compiler. 
 
-homepage         http://jdk.java.net/10/
-master_sites     https://download.java.net/java/GA/jdk10/${version}/19aef61b38124481863b1413dce1855f/13/
-distname         openjdk-${version}_osx-x64_bin
+homepage         http://jdk.java.net/${openjdk_version}/
```

Comment:
OpenJDK 10.0.2 this URL should be http://jdk.java.net/10/, but this resolves to http://jdk.java.net/10.0.2/ which doesn't not exist. Maybe we need a separate `major_version` variable for this? Or we set it separately per port?

Would also be good to use the HTTPS version of this URL while we're at it.